src/me/StevenLawson/TotalFreedomMod/Commands/Command_savinghelp

```diff
@@ -0,0 +1,34 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_IN_GAME)
+@CommandParameters(description = "Clears a player's chat, leave blank to clear public chat.", usage = "/<command> <playername>")
```

Comment:
Incorrect Description and usage. Again, you would know this if there was testing going into place.